keps/sig-node/4622-topologymanager-max-allowable-numa-nodes/kep.yaml

```diff
@@ -5,39 +5,42 @@ authors:
 owning-sig: sig-node
 participating-sigs: []
 status: implementable
-creation-date: "2024-05-08"
+creation-date: "2025-02-15"
 reviewers:
   - "@klueska"
   - "@ffromani"
 approvers:
   - "@sig-node-tech-leads"
```

Comment:
should be @klueska to match the OWNERS file in this directory

Reply:
Thank you for the review. updated.